network/wsPeer.go

```diff
@@ -369,9 +374,9 @@ func (wp *wsPeer) init(config config.Local, sendBufferLength int) {
 		wp.outgoingMsgFilter = makeMessageFilter(config.OutgoingMessageFilterBucketCount, config.OutgoingMessageFilterBucketSize)
 	}
 
-	wp.wg.Add(2)
+	wp.wg.Add(1)
 	go wp.readLoop()
-	go wp.writeLoop()
+	//go wp.writeLoop()
```

Comment:
I was chasing down where the `inSendScheduler` was being read to avoid writing during the `sendLoop`, and found the entire write loop commented out here.

The comment in the `inSendScheduler` is somewhat misleading given that it's only being used to tell if the object is present in some `sendScheduler` and not to indicate wether or not the peer should be handled by the scheduler.

We just flatout disabled the peer writing thread for all peers.

Reply:
If this is intended, we should delete the line, and not comment it out.